chapters/api_concepts.txt

```diff
@@ -242,29 +242,36 @@ brevity:
 [[api_concepts_introspection]]
 === Object Introspection
 
-ANARI supports introspection of objects, i.e., querying supported object
+ANARI supports introspection of objects, i.e., querying supported features, object
 subtypes and information about their parameters, which is particularly useful
 to enumerate and inspect implementation-specific object features. The intended
 use is to allow for building a graphical user interface for, e.g., specific
 renderers or materials, and to provide hints for the debug layer to aid in
-validating features.  Because the meta-information of objects and parameters
```

Comment:
The statement that meta-information is static is useful and should be kept (just the end of the sentence is void).